umbrella-feeds/src/lib.rs

```diff
@@ -456,6 +456,28 @@ fn get_many_price_data<S: HasStateApi>(
     Ok(price_data)
 }
 
+/// View function that returns many price data.
```

Comment:
Please elaborate why this exists in additon to the non-raw one.

Reply:
added additional explanation.